homework-12/todolist.js

```diff
@@ -0,0 +1,37 @@
+window.addEventListener('DOMContentLoaded', () => {
+    const form = document.querySelector('#new-task-form');
+    const input = document.querySelector('#new-task-input');
+    const listEl = document.querySelector('#tasks');
+    form.addEventListener('submit', (event) => {
```

Comment:
Nice to have validation for empty values


https://user-images.githubusercontent.com/84010501/201361157-56c007f2-ca02-49b5-bdb6-404294c81e3f.mov